ansible/roles/dev-desktop/tasks/main.yml

```diff
@@ -11,7 +15,9 @@
       - python3-cryptography
       - quota
       - "linux-modules-extra-{{ vars_linux_kernel_flavor }}"
```

Comment:
Can we also calculate the kernel flavor from `uname -r`? it should be the last item when splitting by `-`.